projects/gitlab.com/gitlab-org/cli/package.yml

```diff
@@ -0,0 +1,38 @@
+distributable:
+  url: https://gitlab.com/gitlab-org/cli/-/archive/v{{version}}/cli-v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  - 1.26.0
+
+provides:
+  - bin/glab
+
+build:
+  script: |
+    go mod download
+    mkdir -p "{{ prefix }}"/bin
+    go build -v -trimpath -ldflags="$LDFLAGS" -o $BUILDLOC ./cmd/glab #FIXME for some reason glab --version returns DEV instead of 1.26.0
+  dependencies:
+    go.dev: ^1.18
+  env:
+    GO111MODULE: 'on'
+    BUILDLOC: '{{prefix}}/bin/glab'
+    LDFLAGS:
+      [-s, -w, "-X=main.Version={{version}}", "-X main.debugMode=false"]
```

Comment:
It'll be this, it varies by go pkg (lol) you may have to look at their CI or look into the source to figure out what key is correct.

Reply:
https://gitlab.com/gitlab-org/cli/-/blob/main/Makefile#L50 >
https://gitlab.com/gitlab-org/cli/-/blob/main/Makefile#L16

Maybe: `build.env.GLAB_VERSION: ${{ version }}`?